### ISSUE TITLE
Research Existing Time-Focussed Apple Watch Apps

### ISSUE THREAD

**Issue by nelsonic** [dwyl/learn-apple-watch-development#3]
@sohilpandya _yesterday_ you sent us a link to the "_Hours_" time tracing app: https://itunes.apple.com/gb/app/hours-time-tracking/id895933956 
![hours-apple-watch-screens-1of2](https://cloud.githubusercontent.com/assets/194400/19472319/b7dfcfd4-951e-11e6-8093-e728a8f9b2a7.png)
![hours-apple-watch-screens-2of2](https://cloud.githubusercontent.com/assets/194400/19472320/b7e454aa-951e-11e6-9f3b-b36f715e8686.png)

This is a _really_ good starting point for our research. 
I am _actively_ using the app on my phone+watch today to test it. 

> **Which _other_ apps** should we be looking at? (_list please_)

**Comment by sohilpandya** [comment]
There are quite a few Time tracking tools out there, but these are the ones that have an accompanying apple watch app. 

- [ ] Timely (free) https://itunes.apple.com/gb/app/timely-simple-time-tracking/id619462420?mt=8
- [ ]  Tyme 2 (£3.99) https://itunes.apple.com/us/app/tyme-2/id1063998403?mt=8
- [ ]  ATracker (free/InAppPurchases) https://itunes.apple.com/us/app/atracker-daily-task-time-tracking/id522008611?mt=8

- Probably worth mentioning the default timer app too, although it doesn't have the tracking features, it's probably the easiest and fastest timer out on the watch.

**Comment by iteles** [comment]
I've also downloaded [Tick](https://www.tickspot.com/) which I'll be trying later this week.

+ [ ] Tick (free for 1 project, monthly subscription for more): https://itunes.apple.com/WebObjects/MZStore.woa/wa/viewSoftware?id=301202863&mt=8

So far, Timely look the most promising from a cursory UX perspective

**Comment by sohilpandya** [comment]
Just been told of Toggl: 

- [ ] Toggl  https://itunes.apple.com/gb/app/toggl-work-time-tracker/id885767775?mt=8

**Comment by nelsonic** [comment]
@sohilpandya yeah, @harrygfox uses Toggl ... might be worth picking his brains on it. 😉

**Comment by sohilpandya** [comment]
Having downloaded and set it up on the laptop yesterday, I went on to download the app for the phone which doesn't have the same option of signing in with google as the online version! (+ no apple watch app) 

I have to say not a great start for Toggl.

**Comment by harrygfox** [comment]
Hey friendos @sohilpandya @nelsonic @iteles 
I've been using this app, Be Focused:
https://xwavesoft.com/be-focused-pro-for-iphone-ipad-mac-os-x.html
I'm finding the pomodoro style time-break function suuuuuuuper useful. I think it's well worth a look at how they arrange the to-do list. You can do pomodoros on a specific task and it tracks the number.

**Comment by sohilpandya** [comment]
Just found this as a new release on the MacOS http://www.fohkuhs.com/
